File: Sakthisri.2534/java/flowcharts/FCP036.java
import java.util.*;
public class FCP036{
public static void main(String[] args){
Scanner input=new Scanner(System.in);
long n=input.nextInt();
long m=input.nextInt();
long max=0;
long min=0;
long r=0;
long lcm=0;
long hcf=0;
if(n>m){
max=n;
min=m;
}
else{
max=m;
min=n;
}
while(min!=0){
r=max%min;
max=min;
min=r;
}
hcf=max;
System.out.println(hcf);
long q=n*m;
lcm=q/hcf;
System.out.println(lcm);
}
}

File: SreeGanesh.2531/Python/FlowchartCode/FCP036.py
m=int(input())
n=int(input())
m1=0
m2=0
if m>n:
    m1=m
    m2=n
else:
    m1=n
    m2=m
lcm=m1
while lcm%m2!=0:
    lcm+=m1
hcf= (m*n)//lcm
print(hcf)
print(lcm)